srcpkgs/yaru-plus/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'yaru-plus'
 pkgname=yaru-plus
-version=4.0
+version=4.1
 revision=2
```

Comment:
Please reset revision to 1.

Reply:
Done, @ericonr. But remember who modified it to 2 were Void Linux memembers. I do not know who did it. I did not accompany the modifications since AUgust.

Reply:
https://github.com/void-linux/void-packages/blob/2dff49c420b27baf6a6dce30891237b9485bfd81/Manual.md#L443-L445